src/components/Button/Button.js

```diff
@@ -11,7 +11,6 @@ import selectors from 'selectors';
 import './Button.scss';
 
 const propTypes = {
-  disable: PropTypes.bool,
```

Comment:
this might cause a bug.
I recently added this so that we can disable the print button when "printing is in progress".

I think we should keep it?

Reasoning:
Looking at the name, it looks like be a general reusuable dumb presentational button component.
It looks like a bit strange that we need to read from the state to know if its disabled or not?

I don't quite know the whole "knowledge" of this file. so let me know if i'm missing something.

Reply:
I checked the commit history and it shows that I added this prop a few months ago. I don't think this prop is used anymore in this component so I just removed it.

I think this is the one that is added by you? https://github.com/PDFTron/webviewer-ui/blob/master/src/components/Input/Input.js#L17

Reply:
ah yes. thanks for the correction.